lib/Math/Matrix.pm6

```diff
@@ -10,9 +11,9 @@ method !column-count { $!column-count }
 subset Positive_Int of Int where * > 0 ;
 
 method new( @m ) {
-    die "Expect an Array of Array" unless all @m ~~ Array;
-    die "All Row must contains the same number of elements" unless @m[0] == all @m[*];
-    die "All Row must contains only numeric values" unless all( @m[*;*] ) ~~ Numeric;
+    fail "Expect an Array of Array" unless all @m ~~ Array;
+    fail "All Row must contains the same number of elements" unless @m[0] == all @m[*];
+    fail "All Row must contains only numeric values" unless all( @m[*;*] ) ~~ Numeric;
```

Comment:
i take it out for now but i dont like routines that can take out the whole app without any way to catch error.